src/services/backend/users-me.js

```diff
@@ -83,7 +83,7 @@ export const authenticateFresh = (username, password) => {
       username,
       password,
     },
-    successCodes: [200],
+    successCodes: [400],
```

Comment:
If this function succeeds on "400 Bad request" response, why you just don't remove it?

Reply:
I just changed that somewhere in the process to work around that and then forgot about it.